### PR TITLE
[14.0] web_widget_child_selector: try to fix image display in README

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Comment by alexis-via** [comment]
The screenshot is not displayed in the README: https://github.com/OCA/web/tree/14.0/web_widget_child_selector

**Comment by pedrobaeza** [comment]
Please put relative paths with .. and the README generator will put the absolute one.

File: web_widget_child_selector/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -4,5 +4,5 @@ Now, you can navigate easily between the hierarchy.
 When we are using readonly, the behaviour is similar than Many2one, but on edition
 we are able to select the parents and childs in order to navigate properly.
 
-.. image:: /web_widget_child_selector/static/description/edit_mode.png
+.. image:: static/description/edit_mode.png
```
Comment (legalsylvain) [inline web_widget_child_selector/readme/DESCRIPTION.rst]:
```suggestion
.. image:: ../static/description/edit_mode.png
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by legalsylvain** [comment]
thanks.

/ocabot merge patch

**Pull request merged**